truss/base/constants.py

```diff
@@ -84,7 +84,7 @@
 
 TRTLLM_SPEC_DEC_TARGET_MODEL_NAME = "target"
 TRTLLM_SPEC_DEC_DRAFT_MODEL_NAME = "draft"
-TRTLLM_BASE_IMAGE = "baseten/briton-server:v0.17.0-fd30ac1"
+TRTLLM_BASE_IMAGE = "baseten/briton-server:v0.17.0-bf10127"
```

Comment:
```suggestion
TRTLLM_BASE_IMAGE = "baseten/briton-server:v0.17.0-6612132"
```
Could you use this image instead? it includes just 2 extra commits which are important for stability.

https://hub.docker.com/layers/baseten/briton-server/v0.17.0-6612132/images/sha256-238f6894cd4d66d3ab9223d1a18af0b9ec8c386446a35101fd765bc077929cf7